src/firefly/java/edu/caltech/ipac/firefly/messaging/Messenger.java

```diff
@@ -33,117 +28,38 @@
  * @version $Id: $
  */
 public class Messenger {
-    public enum Status {CONNECTED, FAIL_TO_CONNECT, NOT_USING};
-    private static final String REDIS_HOST = AppProperties.getProperty("redis.host", "127.0.0.1");
-    private static final int REDIS_PORT = AppProperties.getIntProperty("redis.port", 6379);
-    private static final int MAX_POOL_SIZE = AppProperties.getIntProperty("redis.max.poolsize", 25);
-    private static final String REDIS_PASSWORD = getRedisPassword();
-    private static final Logger.LoggerImpl LOG = Logger.getLogger();
-
-    // message broker..  Jedis
-    private static JedisPool jedisPool;
-    private static Status status = Status.NOT_USING;
+    private static Logger.LoggerImpl LOG = Logger.getLogger();
```

Comment:
When do we use this?

Reply:
Messenger handles messaging. Currently, it handles Firefly’s events exchanged between clients and servers. In the future, we plan to expand its use to other types of events, such as job completion notifications.